apps/sms/views/inbox/js/inbox.js

```diff
@@ -152,12 +162,19 @@ var InboxView = {
     }
   },
 
+  /**
+   * The set of instructions to be executed after the inbox is closed.
```

Comment:
you can be specific here. like what this.cancelEdit does. like leave Inbox Edit Mode